src/components/label/Label.js

```diff
@@ -41,7 +41,7 @@ const Label = ({ text, tooltip, infoTooltip, infoIconProps, showOptionalText, hi
         labelContent.push(
             <InfoIconWithTooltip
                 key="infoTooltip"
-                iconProps={{ className: 'tooltip-icon', ...infoIconProps }}
+                iconProps={{ className: 'bdl-Tooltip-icon', ...infoIconProps }}
```

Comment:
Is this decorating the Tooltip component or the Icon component? If the latter, it should be an icon-specific name.

Reply:
Given that this element lives in `Label`, maybe the class name should be `bdl-Label-infoIcon`. What do you think?

Reply:
Yep, sounds good!